docs/contributor_guide/development_process.rst

```diff
@@ -86,7 +142,8 @@ command
 Note that the Fairlearn dashboard is built using nodejs and requires
 additional steps. To build the Fairlearn dashboard after making changes to it,
 `install Yarn <https://yarnpkg.com/lang/en/docs/install>`_, and then run the
-`widget build script <https://github.com/fairlearn/fairlearn/tree/master/scripts/build_widget.py>`_.
+`widget build script <https://github.com/fairlearn/fairlearn/tree/master/
+scripts/build_widget.py>`_.
```

Comment:
Could you undo this change? I think it's OK to overrun 79 characters for things like URLs which are longer than the limit.